src/test/logical_query_plan/mock_node_test.cpp

```diff
@@ -61,7 +61,7 @@ TEST_F(MockNodeTest, HashingAndEqualityCheck) {
       MockNode::make(MockNode::ColumnDefinitions{{DataType::Int, "a"}, {DataType::Float, "b"}}, "other_name");
   EXPECT_EQ(*_mock_node_b, *_mock_node_b);
   EXPECT_NE(*_mock_node_b, *different_mock_node_1);
-  EXPECT_EQ(*_mock_node_b, *different_mock_node_2);
+  EXPECT_NE(*_mock_node_b, *different_mock_node_2);
```

Comment:
That is because the name wasn't checked in the old version?

Reply:
Correct.